lib/rules/space-unary-ops.js

```diff
@@ -17,12 +17,13 @@ module.exports = function(context) {
     //--------------------------------------------------------------------------
 
     /**
-    * Check if the parent unary operator is "!" in order to know if it's "!!" convert to Boolean or just "!" negation
+    * Check if the node is the first "!" in a "!!" convert to Boolean expression
     * @param {ASTnode} node AST node
-    * @returns {boolean} Whether or not the parent is unary "!" operator
+    * @returns {boolean} Whether or not the node is first "!" in "!!"
     */
-    function isParentUnaryBangExpression(node) {
-        return node && node.parent && node.parent.type === "UnaryExpression" && node.parent.operator === "!";
+    function isFirstBangInBangBangExpression(node) {
```

Comment:
Can you update unittests and add column and line numbers of the errors for this fix then?